Maths/test/AliquotSum.test.ts

```diff
@@ -0,0 +1,5 @@
+import { AliquotSum } from "../AliquotSum";
+
+test.each([[15, 9], [18, 21], [28, 28], [100, 117], [169, 14], [1729, 511], [15625, 3906]])("Aliquot Sum", (num, expected) => {
```

Comment:
consider using the formatting here (`Aliquot Sum of %i is %i`)

Reply:
Makes sense, I'll do that right away